src/GraphQL.Tests/Execution/VariablesTests.cs

```diff
@@ -109,7 +109,7 @@ public void executes_with_complex_input()
               fieldWithObjectInput(input: {a: ""foo"", b: [""bar""], c: ""baz""})
             }
             ";
-            var expected = "{ \"fieldWithObjectInput\": \"{\\\"a\\\":\\\"foo\\\",\\\"b\\\":[\\\"bar\\\"],\\\"c\\\":\\\"baz\\\"}\" }";
+            var expected = "{ \"fieldWithObjectInput\": \"{\\\"a\\\":\\\"foo\\\",\\\"b\\\":[\\\"bar\\\"],\\\"c\\\":\\\"baz\\\",\\\"d\\\":null}\" }";
```

Comment:
These variable tests also seem outdated, so went ahead and updated them. Please verify that this is the expected behaviour.